### PR TITLE
New version: MikeFarah.yq version 4.35.2

### PR DESCRIPTION
**Pull request by Okeanos**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121523)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)  ---

File: manifests/m/MikeFarah/yq/4.35.2/MikeFarah.yq.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+InstallerLocale: en-US
+InstallerType: portable
+Commands:
+- yq
+ReleaseDate: 2023-09-26
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/mikefarah/yq/releases/download/v4.35.2/yq_windows_amd64.exe
+  InstallerSha256: 2E10B644B2FDF7D62354487A075E43FCD7D2F32B972DF713AD5855FD2581EB16
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/m/MikeFarah/yq/4.35.2/MikeFarah.yq.installer.yaml:15]:
```suggestion
- Architecture: x64
  InstallerUrl: https://github.com/mikefarah/yq/releases/download/v4.35.2/yq_windows_amd64.exe
  InstallerSha256: 2E10B644B2FDF7D62354487A075E43FCD7D2F32B972DF713AD5855FD2581EB16
- Architecture: x86
```

Comment (stephengillie) [reply manifests/m/MikeFarah/yq/4.35.2/MikeFarah.yq.installer.yaml:15]: Hi @Okeanos,

It looks like the architectures might have become swapped.

Comment (Okeanos) [reply manifests/m/MikeFarah/yq/4.35.2/MikeFarah.yq.installer.yaml]: D'oh … thanks a lot for pointing that out. Should be fixed now 🤞

**Review by stephengillie — APPROVED** [review]

**Pull request merged**